File: 009/c++/problem9.cc
#include <iostream>
#include <cmath>

using namespace std;

int main() {
  int max = 1000;
  for (int i = 1; i < max; i++) {
    for (int j = i+1; j < max -i; j++) {
      int k = max - i - j;

      //cout << "Numbers : " << i << " " << j << " " << k << endl;
      if (pow(i, 2) + pow(j, 2) == pow(k, 2)) {
        cout << "Product = " << i*j*k << endl;
      }
    }
  }
}

File: 009/python/problem9.py
#! /bin/env python
import sys
max = 1000

for i in range(1,max):
  for j in range(i+1, max-i):
    if (i**2 + j**2 == (max-i-j)**2):
      print("Found: i =", i, "j =", j, "k =", max-i-j)
      print("Product = ",i*j*(max-i-j))
      sys.exit(0)
